app/assets/javascripts/tags.js

```diff
@@ -0,0 +1,70 @@
+$(document).ready(function() {
+  var all_tags = [];
+  var selected_tag_ids = [];
+  $.ajax({
```

Comment:
The tags should only be fetched when the tags page opens, because when the application initially loads there is no `uuid` yet, so the API call will fail.